docs/references/fluxctl.md

```diff
@@ -565,7 +582,7 @@ retrieves from the registry.
    fluxctl --message="Message providing more context for the action" .....
    ```
 
-1. Committer
+2. Committer
```

Comment:
This is not necessary. In Markdown, you can do a numbered list using this:

```md
1. a
1. b
1. c
1. d
```

1. a
1. b
1. c
1. d